lib/logitech_receiver/settings_templates.py

```diff
@@ -118,6 +118,12 @@ def feature_range(name, feature, min_value, max_value,
 
 _SMOOTH_SCROLL = ('smooth-scroll', _("Smooth Scrolling"),
 							_("High-sensitivity mode for vertical scroll with the wheel."))
+_HIRES_INV = ('hires-smooth-invert', _("High Resolution Wheel Invert"),
+							_("High-sensitivity wheel invert mode for vertical scroll."))
+_HIRES_RES = ('hires-smooth-resolution', _("Wheel Resolution"),
+							_("High-sensitivity mode for vertical scroll with the wheel."))
+_HIRES_TGT = ('hires-smooth-hid', _("High Resolution HID++ notification"),
```

Comment:
The way I see is that, if hid-logitech-hidpp driver is loaded, it should switch to HID++ mode, in order to be able to get the resolution at the mouse events. However, if the Kernel is compiled without it, the mouse will produce standard HID events. 

In other words, I think we should merge a patch like this one to the Kernel:
   https://patchwork.kernel.org/patch/9644555/

With such patch, if wheel is producing either HID or HID++ events should be transparent for the user, at least in low resolution mode.

Yet, being able to enable/disable this feature could be useful for debugging. So, perhaps it could be disabled only at GUI - or require some debug flag in order to allow changing it - as I would find useful to be able to dynamically switch it and see if input/evdev is properly reporting events on both modes.